spec/reservation_system_spec.rb

```diff
@@ -0,0 +1,149 @@
+require_relative "spec_helper"
+
+describe 'ReservationSystem class' do
+  describe 'Find room and booking methods' do
+    before do
+      @reservation_system = HotelBooking::ReservationSystem.new('Wyndham', 20)
+    end
+
+    it 'Can find room by booking number' do
+      booking = @reservation_system.make_booking('2001-02-03', '2001-02-07')
```

Comment:
If you are using this `booking` a lot, why not use `let` to initialize it when you need it?